src/meshcat/servers/zmqserver.py

```diff
@@ -128,7 +128,14 @@ def open(self):
         self.bridge.send_scene(self)
 
     def on_message(self, message):
-        pass
+        try:
+            import json
```

Comment:
Let's move this `import` (and the others below) to the top of the file. I'd prefer for all the imports to be unconditional to avoid surprising import errors at run-time.